Source/WebCore/rendering/svg/SVGRenderSupport.h

```diff
@@ -61,7 +61,8 @@ class SVGRenderSupport {
     // Determines whether the passed point lies in a clipping area
     static bool pointInClippingArea(const RenderElement&, const FloatPoint&);
 
-    static void computeContainerBoundingBoxes(const RenderElement& container, FloatRect& objectBoundingBox, bool& objectBoundingBoxValid, FloatRect& strokeBoundingBox, FloatRect& repaintBoundingBox);
+    static void computeContainerBoundingBoxes(const RenderElement& container, FloatRect& objectBoundingBox, bool& objectBoundingBoxValid, FloatRect& repaintBoundingBox, RepaintRectCalculation = RepaintRectCalculation::Fast);
+    static FloatRect computeStrokeBoundingBox(const RenderElement& container);
```

Comment:
Let's call this `computeContainerStrokeBoundingBox` for consistency with `computeContainerBoundingBoxes`.

Reply:
Sounds good, changed!